fastify.d.ts

```diff
@@ -385,6 +385,11 @@ declare namespace fastify {
      * Prints the representation of the internal radix tree used by the router
      */
     printRoutes(): string
+
+    /**
+     * Array that contains all HTTP methods
+     */
+    SUPPORTED_METHODS: [string]
```

Comment:
This should be `string[]` or `Array<string>`. Current syntax would denote an array with just a single string entry

Reply:
Fixed